core/domain/state_domain.py

```diff
@@ -961,7 +962,11 @@ class RecordedVoiceovers(python_utils.OBJECT):
     """
 
     def __init__(self, voiceovers_mapping):
-        """Initializes a RecordedVoiceovers domain object."""
+        """Initializes a RecordedVoiceovers domain object.
+
+        Args:
+            voiceovers_mapping: Recorded Voiceovers. Recorded Voiceovers domain object.
```

Comment:
Please check the type of `voiceovers_mapping`.

Maybe line [1013](https://github.com/oppia/oppia/pull/7741/files#diff-fddf42bff012423be27edc64edd9e4e9L1008) will be helpful?

Reply:
Hi @DubeySandeep , aside from static testing the codes is there some way I could run them so I can check their types. Sorry that's what is hindering my contribution here.

Reply:
You can write a test (or use [the existing test methods](https://github.com/oppia/oppia/blob/develop/core/domain/state_domain_test.py#L1268)) to check the type of translations_mapping and use the run_backend_test.py file t[o run a specific test](https://github.com/oppia/oppia/wiki/Running-Tests#server-side-tests).

OR

You can start the server and follow these steps to create a WrittenTranslation Object:

- Add print statements wherever required
- Login and create an exploration.
- Go to the translation tab and switch to translation mode.
- Write a translation and check the server logs.

Maybe you can find a way to separate the WrittenTranslation class which only depends on the WrittenTranbslation object and create a separate file and run the file, there can be multiple ways to investigate this.

Reply:
Woah thanks.. Gonna look through that now.. I hope this will help my chances for OutReachy?

Reply:
I don't know I didn't see this. I'd follow these steps